notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<ul><li>Please provide truthful and clear limitations of the analysis (and potential biases in data and/or tools)</li><li>Please provide a conclusion for the notebook.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done.</p>

Reply:
closing conversation, resolved